### ISSUE TITLE
changing salesbot response to "seller" and "buyer" instead of "owner" and "winner"

### ISSUE THREAD

**Issue by PonyoAB** [ArtBlocks/artbot#88]
Instead of the salesbot saying "owner" and "winner", what do you think of saying "seller" and "buyer"? "Winner" sounds to my ears like it reinforces the investment angle.
It's a small change for sure (not sure many people would notice!) but I do think it is one of those small things that over time adds up.

**Comment by jakerockland** [comment]
Indeed, that is where that change would live @sukoneck :) we're working on better documenting how the test setup is exposed to external contributors, but if you wanted to send a PR, myself or @ryley-o could pull the changes locally and verify before merging 👍
